src/unpack/fs.ts

```diff
@@ -6,11 +6,13 @@ import { map } from 'streaming-iterables'
 import { CarIndexedReader, CarReader } from '@ipld/car'
 import { CID } from 'multiformats'
 import { UnixFSEntry } from '@vascosantos/ipfs-unixfs-exporter'
+import { FsBlockStore } from '../blockstore/fs'
 
 // tslint:disable-next-line: no-var-requires needs types
 const toIterable = require('stream-to-it')
 
-import { unpack } from './index'
+import { unpack, unpackStream } from './index'
+import { Blockstore } from '../blockstore'
 
 // Node only, read a car from fs, write files to fs
 export async function unpackToFs ({input, roots, output}: {input: string, roots?: CID[], output?: string}) {
```

Comment:
packToFs receives a Blockstore, defaulting to FsBlockstore. Should we do that here too?

Reply:
`unpackToFs` is optimised such that it does not need a blockstore... the user passes the path to the car file on disk so we can use a IndexedCarReader directly, no need for a temp blockstore.